python/lsst/ctrl/stats/dbRecord.py

```diff
@@ -45,7 +45,7 @@ def printValues(self):
             print(mem, "=", value)
 
```

Comment:
Not specific for this line, but I ran flake8 and obtained  for dbRecord.py 

'builtins.str' imported but unused
'builtins.bytes' imported but unused

and in an other file.     /stats/records/record.py 

'builtins.str' imported but unused

do these seem valid ?